### DISCUSSION TITLE
http: invalid host header

### DISCUSSION THREAD

**Discussion by thomas-de-kade** [netbox-community/netbox-docker#1117] [Getting Help]
Hi everyone.
Im new to netbox, im trying to install netbox docker and i'm following this guide: https://github.com/netbox-community/netbox-docker

im trying to create an admin user and get the following error.

sudo docker compose exec netbox /opt/netbox/netbox/manage.py createsupseruser
http: invalid host header

i also dont have the map /opt/netbox/netbox/manage.py

please help, thank you!

**Comment by lestd** [comment]
Why not sh into your container interactively and have a look around?
$ sudo docker exec -it netbox-docker-netbox-1 sh -l
and then 
$ /opt/netbox/netbox/manage.py createsuperuser

Perhaps you may find the manage.py is there after all.
